dev-tools/packaging/templates/docker/Dockerfile.tmpl

```diff
@@ -165,24 +127,49 @@ ENV PATH="$NODE_PATH/node/bin:$PATH"
 # Install the latest version of @elastic/synthetics forcefully ignoring the previously
 # cached node_modules, heartbeat then calls the global executable to run test suites
 # Setup node
+# Also try to add any deps that were missed earlier in terms of playwright reqs
+# playwright install-deps is kept up to date with additional packages chromium needs
+# beyond the base packages listed above
 RUN cd /usr/share/heartbeat/.node \
-  && NODE_DOWNLOAD_URL="" \
-  && case "$(arch)" in \
-       x86_64) \
-           NODE_DOWNLOAD_URL=https://nodejs.org/dist/v${NODE_VERSION}/node-v${NODE_VERSION}-linux-x64.tar.xz \
-           ;; \
-       aarch64) \
-           NODE_DOWNLOAD_URL=https://nodejs.org/dist/v${NODE_VERSION}/node-v${NODE_VERSION}-linux-arm64.tar.xz \
-           ;; \
-       *) \
-           echo >&2 ; echo >&2 "Unsupported architecture \$(arch)" ; echo >&2 ; exit 1 ; \
-           ;; \
-     esac \
-  && mkdir -p node \
-  && curl ${NODE_DOWNLOAD_URL} | tar -xJ --strip 1 -C node \
-  && chmod ug+rwX -R $NODE_PATH \
-  && npm i -g -f @elastic/synthetics && chmod ug+rwX -R $NODE_PATH
+    && NODE_DOWNLOAD_URL="" \
+    && case "$(arch)" in \
+        x86_64) \
+            NODE_DOWNLOAD_URL=https://nodejs.org/dist/v${NODE_VERSION}/node-v${NODE_VERSION}-linux-x64.tar.xz \
+            ;; \
+        aarch64) \
+            NODE_DOWNLOAD_URL=https://nodejs.org/dist/v${NODE_VERSION}/node-v${NODE_VERSION}-linux-arm64.tar.xz \
+            ;; \
+        *) \
+            echo >&2 ; echo >&2 "Unsupported architecture \$(arch)" ; echo >&2 ; exit 1 ; \
+            ;; \
+        esac \
+    && mkdir -p node \
+    && curl ${NODE_DOWNLOAD_URL} | tar -xJ --strip 1 -C node \
+    && chmod ug+rwX -R $NODE_PATH
+
+# Install synthetics as a regular user, installing npm deps as root odesn't work
+RUN chown -R {{ .user }} $NODE_PATH
+USER {{ .user }}
+# If this fails dump the NPM logs
+RUN npm i -g --loglevel verbose -f @elastic/synthetics || sh -c 'tail -n +1 /root/.npm/_logs/* && exit 1'
+RUN chmod ug+rwX -R $NODE_PATH 
+USER root
+
+# Install the deps as needed by the exact version of playwright elastic synthetics uses
+# We don't use npx playwright install-deps because that could pull a newer version
+# Install additional fonts as well
+RUN for iter in {1..10}; do \
+    apt-get update -y && \
+    $NODE_PATH/node/lib/node_modules/@elastic/synthetics/node_modules/.bin/playwright install-deps && \
```

Comment:
This is going to pull in lots of dependencies not related to chromium browser which is the only browser we support right now. Did we check the before/after image size with this change? 

The command needs to be changed to 
```
playwright install-deps chromium
```

to be able to install only dependencies related to the chromium browser.